File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/RoleResVo.java
package org.javamaster.invocationlab.admin.model.erd;

import lombok.Data;

import java.io.Serializable;

@Data
public class RoleResVo implements Serializable {
    private Long id;
    private String roleId;
    private String projectId;
    private String roleName;
    private String roleCode;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/RoleResVo.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

import java.io.Serializable


@AllOpen
class RoleResVo : Serializable {
    var id: Long? = null
    var roleId: String? = null
    var projectId: String? = null
    var roleName: String? = null
    var roleCode: String? = null
}
